src/qt/walletcontroller.cpp

```diff
@@ -46,13 +46,15 @@ std::vector<WalletModel*> WalletController::getWallets() const
     return m_wallets;
 }
 
-std::vector<std::string> WalletController::getWalletsAvailableToOpen() const
+std::map<std::string, bool> WalletController::getAllWallets() const
```

Comment:
nit: could be more self-documenting - something like `getWalletsNamesAndLoaded`

Reply:
Renamed these methods and added comments, hope it's clear now.